src/server.js

```diff
@@ -4,7 +4,7 @@ dotenv.config();
 import { Telegraf } from "telegraf";
 import express from "express";
 const app = express();
-const port = 3000;
+const port = process.env.PORT || 3000;
 
 app.get("/", (req, res) => {
   res.send("Yes, the bot server is running at this address!");
```

Comment:
Should we change endpoint to ```/health```
Should we return 2xx status code instead of sending this message?

Reply:
sure, may I ask why? as this end point or our web url is not visible to any user. We'll have a separate end point for webhooks as when we work on #22. Any endpoint other than that is not useful other than telling just devs that "this server is up"

unless i am missing something. lmk

Reply:
This is for devs to know that all services are up and running. For example, db service, webhooks, etc. but now that I think about it, we should probably have different rest end point for that apart from this for health check.

Reply:
really good idea. server status and diagnostics